editor/src/components/canvas/controls/grid-controls.tsx

```diff
@@ -1848,3 +1857,9 @@ export function controlsForGridPlaceholders(gridPath: ElementPath): ControlWithP
     priority: 'bottom',
   }
 }
+
+function sceneTopOffset(canvasScale: number) {
+  const labelHeight = document.getElementById(SceneLabelTestID)?.getBoundingClientRect().height ?? 0
```

Comment:
I don't remember if there was a reason why we didn't give the scene label an explicit height (I just checked assuming we would have). I think it would make sense to do that and then export it as a const rather than querying the DOM here

Reply:
I thought so too, but now I see its offset is calculated based on the font size, so I did not want to change that. But of course I see the point so maybe it's worth simplifying that whole thing and just give it an explicit height.

Reply:
done in [`74a7c44` (#6486)](https://github.com/concrete-utopia/utopia/pull/6486/commits/74a7c44bae86e65d98b25598ac309b5c0dce1a6c)

Reply:
Thank you!